packages/app-types/index.d.ts

```diff
@@ -36,13 +41,28 @@ export class FirebaseApp {
    */
   options: FirebaseOptions;
 
+  /**
+   * The settable config flag for GDPR opt-in/opt-out
+   */
+  automaticDataCollectionEnabled: boolean;
+
   /**
    * Make the given App unusable and free resources.
    */
   delete(): Promise<void>;
 }
 
 export interface FirebaseNamespace {
+  /**
+   * Create (and initialize) a FirebaseApp.
+   *
+   * @param options Options to configure the services used in the App.
+   * @param config The optional config for your firebase app
+   */
+  initializeApp(
+    options: FirebaseOptions,
+    config?: FirebaseAppConfig
```

Comment:
Can you define this as `string|FirebaseAppConfig`

Reply:
We can, but Typescript supports overloading function definitions so I opted to do that instead.